console/console-init/ui/src/modules/project/dailogs/CreateProject/CreateProject.tsx

```diff
@@ -414,18 +418,26 @@ const CreateProject: React.FunctionComponent = () => {
                 <Button
                   variant="primary"
                   type="submit"
-                  id="submit-btn"
+                  id="c-project-next-button"
```

Comment:
This is submit button not next and use  prefix create instead of c.
id="create-project-submit-button"